actions/access_keys/count.go

```diff
@@ -14,15 +14,14 @@ import (
 // @Description	Count of access keys
 // @Tags		Access-key
 // @Produce		json
-// @Param		metadata query string false "metadata"
-// @Param		conditions query string false "conditions"
+// @Param		CountRequestParameters body actions.CountRequestParameters false "CountRequestParameters model containing metadata and conditions"
```

Comment:
Maybe something like:
```suggestion
// @Param		CountRequestParameters body actions.CountRequestParameters false "Enables precise filtering of resource counts using custom conditions or metadata, catering to specific business or analysis needs"
```

For all count requests body.

Reply:
Done